src/test/test.cpp

```diff
@@ -271,68 +266,11 @@ const lest::test specification[] = {
 				};
 				uint32_t k = 2;
 				auto means = dkm::kmeans_lloyd(data, k);
-				double inertia = dkm::means_inertia(data, means);
+				double inertia = dkm::means_inertia(data, means, k);
 				EXPECT(284.256926 == lest::approx(inertia).epsilon(1e-6));
 			}
 		}
 	},
-
-	CASE("Test dkm::get_best_means",) {
-		SETUP() {
-			std::vector<std::array<double, 2>> points{
-				{66.01742226,  48.70477854},
-				{62.30094932, 108.44049522},
-				{39.60740312,  12.07668535},
-				{35.57096194,  -7.10722525},
-				{39.90890238,  61.89509695},
-				{27.5850295 ,  85.50226002},
-				{51.14012591,  27.90650051},
-				{58.6414776 ,  31.97020798},
-				{14.75127435,  69.36707669},
-				{73.66255253,  84.73455103},
-				{-1.31034384,  66.10406579},
-				{41.91865987,  56.5003107 },
-				{33.31116528,  45.92203855},
-				{57.12362692,  37.73753163},
-				{ 2.68915431,  51.35514789},
-				{39.76543196,  -5.99499795},
-				{72.64312341,  61.43756623},
-				{30.97140948,  29.49960625},
-				{25.31232669,  35.88059477},
-				{57.67046396,  35.05019015}
-			};
-			SECTION("Invoke dkm k-means with 5 clusters 10 separate times") {
-				std::vector<std::tuple<std::vector<std::array<double, 2>>, std::vector<uint32_t>>> means_list;
-				for (size_t i = 0; i < 10; ++i) {
-					means_list.push_back(dkm::kmeans_lloyd(points, 5));
-				}
-
-				// best means via dkm_utils
-				auto best_means = dkm::get_best_means(points, means_list);
```

Comment:
Why delete this test entirely instead of modifying it to work with the new get_best_means function?